src/pocketmine/entity/object/FallingBlock.php

```diff
@@ -70,6 +81,10 @@ protected function initEntity(CompoundTag $nbt) : void{
 		$damage = $nbt->getByte("Data", 0);
 
 		$this->block = BlockFactory::get($blockId, $damage);
+		if ($this->block instanceof Anvil) {
+			$this->behaviours = [new DestroyWhileFalling($this,	[Block::TORCH, Block::COLORED_TORCH_RG, Block::COLORED_TORCH_BP,
```

Comment:
Not a fan of this. This should be a property of the blocks themselves.

Reply:
There's also a rogue tab in the argument list here.

Reply:
There is no property system for blocks in PMMP so i would have to generate a getter for it. Also this would not allow for any dynamic lists (since the behaviour is then dependent on the block property) of destoryable blocks.